blocks/product-support/product-support.js

```diff
@@ -27,6 +29,12 @@ function getAssets(product, type, allType) {
   return product.assets.filter((asset) => (type === allType || asset.Type === type));
 }
 
+function parseYoutubeCode(url) {
```

Comment:
just cosmetic, would extract regex as constant and use more descriptive variable names:
```
const YOUTUBE_URL_REGEX = /^.*((youtu.be\/)|(v\/)|(\/u\/\w\/)|(embed\/)|(watch\?))\??v?=?([^#&?]*).*/;

function parseYoutubeCode(url) {
   const matchUrl = url.match(YOUTUBE_URL_REGEX);
   const videoCode = matchUrl && matchUrl[7];
   
   return (videoCode && videoCode.length === 11) ? videoCode : false;
}
```